src/ripple/rpc/handlers/LedgerEntry.cpp

```diff
@@ -71,7 +71,12 @@ doLedgerEntry(RPC::JsonContext& context)
     {
         expectedType = ltCHECK;
 
-        if (!uNodeIndex.parseHex(context.params[jss::check].asString()))
+        if (!context.params[jss::check].isString() && context.apiVersion > 1u)
```

Comment:
Is this the only condition in which params are invalid?

Reply:
thanks, did a full test and there were other fields that needed to be fixed too